src/layout/layout.ts

```diff
@@ -39,7 +39,7 @@ type AllComponents = ComponentTypeConfigs[CompTypes]['layout'];
  *  const myImageComponent:CompExternal<'Image'> = ...
  *
  * @see CompInternal
- * @see LayoutNode
+ * @see REMOVELayoutNode
  */
 export type CompExternal<Type extends CompTypes = CompTypes> = Extract<AllComponents, { type: Type }>;
```

Comment:
What is `REMOVELayoutNode`?

Reply:
Whoops! :sweat_smile: I tried an approach where I renamed one of the types and thought I'd manually go trough cases where I couldn't automatically have the IDE rewrite to use the new unified type. I abandoned that idea after a short while, but didn't notice it changed this reference. :grimacing: Thanks!